tutorials/authentication.mdx

```diff
@@ -558,7 +560,13 @@ async fn post_signup(
 }
 let session_token = new_session(database, random, user_id);
 ```
-
+We also have to create the Sessions table using SQL, the code is as follows:
+```bash
+Create Table Sessions(
+        id VarCHAR(6) PRIMARY KEY,
+        session_token VArChar(6)
+);
+```
```

Comment:
This sql should be like it is in the source code: https://github.com/kaleidawave/axum-shuttle-postgres-authentication-demo/blob/main/schema.sql

I would move this just below "### Adding sessions to signup", before the code showing how post a session to the sessions table.